src/api/routes/articles.js

```diff
@@ -0,0 +1,13 @@
+import { Router } from 'express';
+import checkAuth from '../../middlewares/checkAuth';
+import ArticleController from '../controllers/articleController';
+import errorHandler from '../../middlewares/errorHandler';
+import uploadImage from '../../middlewares/upload';
+
+const articleRouter = new Router();
+
+articleRouter
+  .route('/articles')
```

Comment:
as we'll all use this router, it's better to use this route in the `index.js` then use `.post(checkAuth, uploadImage.single('image')`